app/views/tag/_tagging.html.erb

```diff
@@ -1,4 +1,12 @@
-<h1>Tags</h1>
+<!-- This is the sidebar tagging display, also renders Subscribe button for multiple subscription -->
+<div>
+  <h1 style="display: inline-block;">Tags</h1>
+  <% if @node %>
```

Comment:
Is there any way we can refactor this if-else. I don't think there is a way. Do you think we can do it like we did last time?

Reply:
Which way last time? Actually this is how it was originally so that's why I didn't want to tamper the original code

Reply:
Ok no problem. 
I made this resolved.